### DISCUSSION TITLE
Downloading LXC template seeming takes for ever, not sure if even working?

### DISCUSSION THREAD

**Discussion by bobloadmire** [tteck/Proxmox#2995] [General]
Installing Sonarr LXC, got to Downloading LXC Template, its been 20 minutes, still running. Anything I can do to speed up the download, or manually upload a debian iso?

downloading http://download.proxmox.com/images/system/debian-12-standard_12.2-1_amd64.tar.zst to /var/lib/vz/template/cache/debian-12-standard_12.2-1_amd64.tar.zst
--2024-05-16 14:36:50--  http://download.proxmox.com/images/system/debian-12-standard_12.2-1_amd64.tar.zst
Resolving download.proxmox.com (download.proxmox.com)... 170.130.165.90, 2a0b:7140:8:100::90
Connecting to download.proxmox.com (download.proxmox.com)|170.130.165.90|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 126129049 (120M) [application/octet-stream]
Saving to: '/var/lib/vz/template/cache/debian-12-standard_12.2-1_amd64.tar.zst.tmp_dwnl.207656'
     0K ........ ........ ........ ........ 26%  121K 12m25s
 32768K ........ ........ ........ ........ 53%  125K 7m48s

**Comment by tteck** [comment]
Looks like you got some kind of network issues. 🤷🏻‍♂️

**Comment by bobloadmire** [comment]
hmm well it finally finished. speedtest has me at 500/500 🤷‍♂️

**Comment by opsdave** [comment]
I've look like same situation. Downloading CentOs container, take long time to start to download.

[...]
HTTP request sent, awaiting response... 200 OK
Length: 104646080 (100M) [application/octet-stream]
Saving to: '/var/lib/vz/template/cache/centos-9-stream-default_20221109_amd64.tar.xz.tmp_dwnl.99776'

[Its start to download after 15 minutes]

     0K ........ ........ ........ ........ 32% 84.0K 13m47s
...

There is something to do to fix it?
